apollo-router/src/spec/schema.rs

```diff
@@ -116,8 +116,21 @@ impl Schema {
                 if url.is_empty() {
                     return Err(SchemaError::MissingSubgraphUrl(name.to_string()));
                 }
-                let url = Uri::from_str(url)
-                    .map_err(|err| SchemaError::UrlParse(name.to_string(), err))?;
+                // there is noi standard for unix socket URLs apparently
+                let url = if url.starts_with("unix://") {
+                    println!("authority: {}", &url[7..]);
```

Comment:
```suggestion
```

Reply:
fixed in [512b54b](https://github.com/apollographql/router/pull/4757/commits/512b54b3a36c6f13168081965eb6a33052b0d482)